draft-ysl-cats-metric-definition.md

```diff
@@ -94,67 +90,72 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 # Definition of Metrics
 
-Definition and usage of specific metrics are related to the intended use case. However, when considering disseminating compute metrics to network devices, appropriate categorization and abstraction of CATS metrics is required in order to avoid introducing extra complexity into the network.
-
-This document defines three abstract metric levels to meet different requirements use cases listed in {{!I-D.ietf-cats-usecases-requirements}}:
+Introducing a definition of metrics necessitates balancing the following trade-off: if the metrics are too fine-grained, they become unscalable due to the excessive number of metrics that must be communicated through the protocol. Conversely, if the metrics are too coarse-grained, they may lack the necessary information to make informed decisions. To ensure scalability while providing sufficient detail for effective decision-making, we propose a definition of metrics that incorporates three levels of abstraction:
```

Comment:
Thanks for your comments Chen. 

Is this comment just a question of style? If so, this one may seem better:

"Introducing a definition of metrics requires balancing the following trade-off:"

Reply:
it looks good to me. I learn from Sue and other experts in IETF that we should use simple and clear words in writing standards, because the purpose of a standard is to let people understand easily and implement it :)